TLM/TLM/State/ConfigData/Main.cs

```diff
@@ -50,20 +52,28 @@ public class Main {
 		/// </summary>
 		public bool ShowCompatibilityCheckErrorMessage = false;
 
-        /// <summary>
-        /// Shows warning dialog if any incompatible mods detected
-        /// </summary>
-        public bool ScanForKnownIncompatibleModsAtStartup = true;
+		/// <summary>
+		/// Shows warning dialog if any incompatible mods detected
+		/// </summary>
+		public bool ScanForKnownIncompatibleModsAtStartup = true;
 
-        /// <summary>
-        /// Skip disabled mods while running incompatible mod detector
-        /// </summary>
-        public bool IgnoreDisabledMods = true;
+		/// <summary>
+		/// Skip disabled mods while running incompatible mod detector
+		/// </summary>
+		public bool IgnoreDisabledMods = true;
+
+		/// <summary>
+		/// Prefer Miles per hour instead of Kmph (affects speed limits display
+		/// but internally Kmph are still used).
+		/// </summary>
+		public bool DisplaySpeedLimitsMph = false;
 
 		public void AddDisplayedTutorialMessage(string messageKey) {
-			HashSet<string> newMessages = DisplayedTutorialMessages != null ? new HashSet<string>(DisplayedTutorialMessages) : new HashSet<string>();
+			HashSet<string> newMessages = DisplayedTutorialMessages != null
```

Comment:
I disagree with the formatting for your one-line if statements. I think they're better as, well, one-line.